### PR TITLE
Spring 2016 updates

### PR DESCRIPTION
**Pull request by weronica**
- New staff info (i.e., names, email addresses, photos, office hours, labs). 
- Removed Spring 2016 content.
- Added Fall 2016 content.
- Some small formatting fixes/changes.
- Updated readme with site installation/run instructions.

r @feefles @ballerlina

### CODE REVIEW

**Comment by feefles** [comment]
We should update the regrade policy if we are indeed switching to using docs (which I think we should), and check that the late policy is correct re: using 2 late days at a time.

File: _data/staff.yaml [hunk 1]
```diff
@@ -1,125 +1,151 @@
...
-  email: davix@seas
-  lab: 206 (Monday 5--6)
-  office_hours: Wednesday 7--9
+  office_hours: TBA
 
 - name: Fifi Yeung
-  pic: fifi.png
-  email: eyeung@seas
+  pic: eyeung.png
   extra_title: Head TA
```
Comment (feefles) [inline _data/staff.yaml]: But I'm not a head anymore 🐙

File: _data/labs.yaml [hunk 2]
```diff
@@ -1,76 +1,6 @@
-- date: 2016-01-25
+- date: 2016-08-12
```
Comment (feefles) [inline _data/labs.yaml]: This doesn't seem right...

File: _data/staff.yaml (hunk 1 above)
Comment (weronica) [inline _data/staff.yaml]: Whoops yes, you're right. Could you update that and reorder yourself
alphabetically by first name?

File: _data/labs.yaml (hunk 2 above)
Comment (weronica) [inline _data/labs.yaml]: Whoops yes, should be 2016-09-12.

File: _data/staff.yaml (hunk 1 above)
Comment (weronica) [inline _data/staff.yaml]: Haha what was the alternate title we were brainstorming for you? 😉

**Pull request merged**